parm/archive/gdasocean.yaml.j2

```diff
@@ -0,0 +1,9 @@
+gdasocean:
+    {% set head = "gdas.ocean.t" + cycle_HH + "z." %}
+    name: "GDASOCEAN"
+    target: "{{ ATARDIR }}/{{ cycle_YMDH }}/gdasocean.tar"
+    required:
+        {% for fhr in range(FHMIN, FHMAX + 1, FHMAX) %}
+        - "{{ COM_OCEAN_HISTORY | relpath(ROTDIR) }}/{{ head }}inst.f{{ '%03d' % fhr }}.nc"
```

Comment:
@AndrewEichmann-NOAA which forecast hours are supposed to be archived here?  

Currently we are getting: 
gdas.20210324/12/model_data/ocean/history/gdas.ocean.t12z.inst.f000.nc
gdas.20210324/12/model_data/ocean/history/gdas.ocean.t12z.inst.f009.nc

and not the f003 and f006, is that okay or should those forecast hours also be archived?  I believe with your fix in https://github.com/NOAA-EMC/global-workflow/pull/2594 f003 and f006 would also be archived.

Reply:
Yeah, I think that should be `FHOUT_OCNICE`, not `FHMAX`, for the step size in the for loop.  I'll update and test.